### ISSUE TITLE
9.5.3 / 9.5.4 Bug - Missing financial data with computer template 

### ISSUE THREAD

**Issue by mirda123** [glpi-project/glpi#8953]
Describe the bug

When creating a computer using a template - financial data is added BUT no financial data is saved in the computer after choosing "add"

To reproduce

glpi 9.5.3 and glpi 9.5.4

1 - add a computer
2 - choose a template with financial data
3 - look at the financial data : OK
4- Save the new computer created = " add"
4 - open the computer : financial data is empty.

I  tried to disable all plug in ==> same 

Expected behavior

Financial data...

**Logs**

will try ti get them

```
paste logs here
Find them in *-error.log files under glpi/files/_log/
```

**Screenshots**

![bug01](https://user-images.githubusercontent.com/78594318/114387244-7e370e80-9b92-11eb-9d7f-61cc647d822a.jpg)
![bug02](https://user-images.githubusercontent.com/78594318/114387257-81ca9580-9b92-11eb-9569-b3772c4d255f.jpg)

**Your GLPI setup (you can find it in Setup > General menu, System tab)**

Operating system: Linux xxxxxxx   4.4.0-104-generic #127-Ubuntu SMP Mon Dec 11 12:16:42 UTC 2017 x86_64
PHP 7.4.12 apache2handler (Core, FFI, PDO, Phar, Reflection, SPL, SimpleXML, Zend OPcache, apache2handler, apc, apcu, bz2,
	calendar, ctype, curl, date, dom, exif, fileinfo, filter, ftp, gd, gettext, hash, iconv, imap, intl, json, ldap, libxml,
	mbstring, mysqli, mysqlnd, openssl, pcre, pdo_mysql, posix, readline, session, shmop, sockets, sodium, standard, sysvmsg,
	sysvsem, sysvshm, tokenizer, xml, xmlreader, xmlrpc, xmlwriter, xsl, zip, zlib)
Setup: max_execution_time="30" memory_limit="128M" post_max_size="8M" safe_mode="" session.save_handler="files"
	upload_max_filesize="2M" 
Software: Apache/2.4.18 (Ubuntu) (Apache/2.4.18 (Ubuntu) Server at xxxxxxx   Port 443)
	Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:87.0) Gecko/20100101 Firefox/87.0
Server Software: Ubuntu 16.04
	Server Version: 10.0.38-MariaDB-0ubuntu0.16.04.1
	Server SQL Mode: 
	Parameters: 
	Host info: Localhost via UNIX socket

**Additional context**

plug- in all disabled ==> same problem
have tried with super-admin profile, technician profile... ==>  same problem ...

**Comment by mirda123** [comment]
i'm not the only on with the problem : 

https://github.com/glpi-project/glpi/issues/8544

**Comment by cedric-anne** [comment]
Hi,

I cannot reproduce on current state of 9.5/bugfixes branch. Can you try to see if problem has been solved since 9.5.4 release ?

Regards

**Comment by mirda123** [comment]
thanks for your answer, i'm searching on github if i can find another similar problem already solved ...  so far nothing except https://github.com/glpi-project/glpi/issues/8544, but maybe with wrong keyword ..    : (  
how else can i find if it's solved ? is there a 9.5.5 beta ?

**Comment by cedric-anne** [comment]
> thanks for your answer, i'm searching on github if i can find another similar problem already solved ... so far nothing except #8544, but maybe with wrong keyword .. : (
> how else can i find if it's solved ? is there a 9.5.5 beta ?

You can get the latest nightly build here: https://github.com/glpi-project/glpi/suites/2470766824/artifacts/53125932

**Comment by mirda123** [comment]
Hi ! I found the bug !

problem occurs only when general setup is: default active financial data = YES 
if you change to NO , financial data are OK when creating a computer...
if i go back to NO , financial data disapper when creating...  

can you reproduce the bug  ?

**Comment by CireSacud26** [comment]
> 
> 
> Hi ! I found the bug !
> 
> problem occurs only when general setup is: default active financial data = YES
> if you change to NO , financial data are OK when creating a computer...
> if i go back to NO , financial data disapper when creating...
> 
> can you reproduce the bug ?
> configuration / generale / parc / activer les informations financières et administratives par defaut : switch to NO : solved
> configuration / generale / parc / activer les informations financières et administratives par defaut : switch to YES : bug (9.5.3 and 9.5.4)

Hi !

I confirm the bug and this behavior with GLPI 9.5.4.

**Comment by mirda123** [comment]
Hi, pb is still there ! 
waiting for fix

**Comment by mirda123** [comment]
Hi, pb is still there !
waiting for a fix thanks

**Comment by mirda123** [comment]
Hi, pb is still there !
waiting for fix

**Comment by cedric-anne** [comment]
Hi,

Should be fixed by #10406.

Regards
